exonum/src/node/state.rs

```diff
@@ -352,8 +352,8 @@ impl BlockState {
     }
 
     /// Returns the changes that should be made for block committing.
-    pub fn patch(&self) -> &Patch {
-        &self.patch
+    pub fn patch(&mut self) -> Patch {
+        self.patch.take().unwrap()
```

Comment:
Is the `unwrap` is really safe here ?

Reply:
Yes, because this patch must be used only once. I change `unwrap` to `expect` for clarity.